server/migrations/20240325205951-cleanupDuplicatesAndRegenerateHashes.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const { regenerateResultsAndRecalculateHashes } = require('./utils');
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        return queryInterface.sequelize.transaction(async transaction => {
+            // Remove unnecessarily created test plan versions for ALL 35 test
+            // plan versions for commit,
+            // https://github.com/w3c/aria-at/commit/836fb2a997f5b2844035b8c934f8fda9833cd5b2
+            // None of the following ever left R&D
+            await queryInterface.sequelize.query(
+                `delete
+                 from "TestPlanVersion"
+                 where id in
+                       (55591, 55592, 55593, 55594, 55595, 55596, 55597, 55598, 55599, 55600, 55601, 55602, 55603, 55604, 55605, 55606,
+                        55607, 55608, 55609, 55610, 55611, 55612, 55613, 55614, 55615, 55616, 55617, 55618, 55619, 55620, 55621, 55622,
+                        55623, 55624, 55625)`,
+                {
+                    transaction
+                }
+            );
+
+            // Reset the phases and remove deprecatedAt dates for versions affected by above wrongly created versions:
+            // checkbox-tri-state
+            // link-css
+            // link-img-alt
+            // main
+            // meter
+            // radiogroup-roving-tabindex
+            // slider-multithumb
+            // switch
+            await queryInterface.sequelize.query(
+                `update "TestPlanVersion"
+                 set phase          = 'RD',
+                     "deprecatedAt" = null
+                 where id in (1566, 1591, 1618, 1798, 2129, 2162, 2245, 2347)`,
```

Comment:
Again here, this feels like it needs some secondary confirmation that this is the correct update to apply on these rows